charts/opensearch/templates/statefulset.yaml

```diff
@@ -365,6 +365,12 @@ spec:
           #!/usr/bin/env bash
           set -euo pipefail
 
+          {{- range $plugin := .Values.plugins.removeList }}
+          if ./bin/opensearch-plugin list | grep -q {{ $plugin }}; then
```

Comment:
Why is there a different behavior b/w os and osd plugin removal?
Is it necessary to do a check before OS removal but not OSD?

Reply:
Thanks for catching this, we should check if the plugin installed before remove it for both OS and OSD